resources/views/components/exchanges/filter-dropdown.blade.php

```diff
@@ -0,0 +1,16 @@
+@props(['options', 'selected', 'icon', 'param'])
+
+<x-exchanges.dropdown
+    :title="trans($options[$selected])"
+    :icon="$icon"
+>
+    @foreach ($options as $value => $label)
+        @if ($value === $selected)
+            @continue
+        @endif
+
```

Comment:
according to designs, we show and highlight the selected option :thinking: 

Design:
![image](https://github.com/ArdentHQ/arkscan/assets/8069294/4f06a37d-4dce-4cb2-a341-cba4f03148af)

Imp:
![image](https://github.com/ArdentHQ/arkscan/assets/8069294/c4775717-24b1-47bc-b5dd-0db0192f3a60)